templates/primary-secondary/test/primary-secondary.visual-diff.js

```diff
@@ -71,6 +78,9 @@ describe('d2l-template-primary-secondary', () => {
 				if (test.options.position) {
 					const pos = test.options.position;
 					await moveDivider(page, sel, pos.dir, pos.steps);
+					if (test.options.resetDir) { // Needed for retries
```

Comment:
I tried to label when things were only being done for retries, so that when we go back to rewrite these tests with a fresh fixture each time, we aren't adding unnecessary steps